client/package.json

```diff
@@ -33,7 +33,7 @@
     "react-router-dom": "^6.10.0",
     "react-sweet-state": "^2.7.1",
     "scheduler": "^0.23.0",
-    "stripe": "^13.7.0",
+    "stripe": "^14.22.0",
```

Comment:
this should be installed in the server workspace instead?

Reply:
Reverted back to master. Updated server workspace.